packages/client-twitter/src/interactions.ts

```diff
@@ -282,6 +290,37 @@ export class TwitterInteractionClient {
             this.client.saveRequestMessage(message, state);
         }
 
+        const pizzaCheck = `
+        You are checking to see if someone is asking you to order a pizza.
+        They should explicitly ask for a pizza order.
+
+        Here is the tweet they posted:
+        ${currentPost}`
+        + pizzaDecisionFooter;
+
+        const pizzaCheckResponse = await generateText({
```

Comment:
Highly recommend using generateObjectV2 which returns a typesafe JSON instead of having the parse the response and hope for the best

Reply:
generateObjectV2 doesn't work with Groq, it throws "Error: Unknown model at getEncodingNameForModel". Is that a known thing? 
in generateText is hardcoded to gpt-4o